packages/core/src/app/customer/StripeGuestForm.tsx

```diff
@@ -0,0 +1,246 @@
+import { CustomerInitializeOptions, CustomerRequestOptions } from '@bigcommerce/checkout-sdk';
+import { withFormik, FieldProps, FormikProps } from 'formik';
+import React, { memo, useCallback, useEffect, useState, FunctionComponent, ReactNode } from 'react';
+import CheckoutStepStatus from '../checkout/CheckoutStepStatus';
+import { getAppliedStyles } from '../common/dom';
+
+import { TranslatedString } from '../locale';
+import { PrivacyPolicyField } from '../privacyPolicy';
+import { Button, ButtonVariant } from '../ui/button';
+import { BasicFormField, Fieldset, Legend } from '../ui/form';
+import { LoadingOverlay } from '../ui/loading';
+
+import { GuestFormValues } from './GuestForm';
+import SubscribeField from './SubscribeField';
+
+export interface StripeGuestFormProps {
```

Comment:
Please can we review the interface here and only pass what's needed for stripe link component.